packages/sdk/src/conversation/middleware.ts

```diff
@@ -120,33 +132,44 @@ export class CommandResponseMiddleware implements Middleware {
   }
 
   private classifyActivity(activity: Activity): ActivityType {
-    if (this.isCommandReceived(activity)) {
-      return ActivityType.CommandReceived;
+    if (activity.type === ActivityTypes.Message) {
+      return ActivityType.CurrentBotMessaged;
     }
 
     return ActivityType.Unknown;
   }
 
-  private isCommandReceived(activity: Activity): boolean {
-    if (this.commandHandlers) {
-      const commandText = this.getActivityText(activity);
-      const handlers = this.filterCommandHandler(commandText, this.commandHandlers);
-      return handlers.length > 0;
+  private matchPattern(pattern: string | RegExp, text: string): boolean | RegExpMatchArray {
+    const normalizedText = text?.trim().toLocaleLowerCase();
```

Comment:
The same as line 164. Since both are private functions, text can be normalized at the very beginning.

Reply:
make sense, all these normalization can be removed since this is already handled in `getActivityText()`.